### PR TITLE
Updated Kusto

### PR DESCRIPTION
**Pull request by oflipman**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

this version has a breaking change, hot_cache_period and soft_delete_period formats.
But it is our first version of GA

### CODE REVIEW

**Comment by oflipman** [comment]
@tjprescott.
Can some one please review this PR?

File: src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py [hunk 1]
```diff
@@ -6,17 +6,19 @@
...
+)
 
 
 def load_arguments(self, _):
 
     # Kusto clusters
     sku_arg_type = CLIArgumentType(help='The name of the sku.',
                                    arg_type=get_enum_type(AzureSkuName))
-    time_format_explenation = 'Time format is DDDD:HH:MM:SS (for example, 10 years would be 3650:00:00:00).'
+    time_format_explenation = 'Duration in ISO8601 format (for example, 100 days would be P100D).'
```
Comment (tjprescott) [inline src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py]: There's no bump in the SDK version here. What format does the service actually require?

File: src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/tests/latest/test_kusto_commands.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def test_kusto_cluster_stop_start(self, resource_group):
 
         self.cmd('az kusto cluster stop -n {name} -g {rg}')
 
-        # Verify that the state is stopped
+        # Verify that the state is stop ped
```
Comment (tjprescott) [inline src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/tests/latest/test_kusto_commands.py]: typo

File: src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/custom.py [hunk 3]
```diff
@@ -2,15 +2,26 @@
...
 
+class AzureSkuName(Enum):
+
+    d13_v2 = "D13_v2"
+    d14_v2 = "D14_v2"
+    l8 = "L8"
+    l16 = "L16"
+    d11_v2 = "D11_v2"
+    d12_v2 = "D12_v2"
+    l4 = "L4"
```
[the remark this answers is not in the document]
Comment (oflipman) [reply src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/custom.py]: We are in a proccess to remove some of the SKUs, and we don't want to allow users to choose them.
that is why I could not get it from the SDK.

File: src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py (hunk 1 above)
Comment (oflipman) [reply src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py]: There is a bump in the SDK, this PR uses version 2019-01-21, while the current code uses version 2018-09-07-preview.
In the new SDK, the type of the duration is ISO8601

File: src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/custom.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (oflipman) [reply src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/custom.py]: I will add TODO (delete them and use the python library object when we publish our new version of python clients)

**Pull request merged**